Additional website docs updates (#2344)
open-telemetry/opentelemetry-js: website_docs/getting_started/browser.md

weight: 2
---

This guide uses the example application in HTML & javascript provided below, but the steps to instrument your own application should be broadly the same.

- [Example Application](#example-application)
  - [Installation](#installation)
  - [Initialization and Configuration](#initialization-and-configuration)
  - [Creating a Tracer Provider](#creating-a-tracer-provider)
  - [Creating an Exporter](#creating-an-exporter)

>   - [Add Plugins](#add-plugins)

Reviewer: Add instrumentations?

</html>
```

### Installation

> To create traces in the browser, you will need `@opentelemetry/web`, and the plugin `@opentelemetry/plugin-document-load`:

Reviewer: Shouldn't this be `@opentelemetry/instrumentation-document-load` also should we replace mentions of plugin to instrumentation?

Author: Good catch!

> npm install --save @opentelemetry/web @opentelemetry/instrumentation-document-load @opentelemetry/context-zone

Author: > I know this isn't a part of your PR, but wanted to point it out. I actually ran into this issue while following the example bc I was on npm 6. I think the simplest thing is to just install it as a dep and gloss over it not being needed on v7. 

For me having a documentation that makes it easy for everyone to get started is essential, so that's definitely within the scope of this PR! I think having it installed explicitly is fine, I commit your suggestion. thanks

  ]
}
```

### Add Plugins

If you want to instrument AJAX requests, User Interactions and others, you can add plugins to your
instrumentation:

```javascript
registerInstrumentations({
  instrumentations: [
    new UserInteractionInstrumentation(),
    new XMLHttpRequestInstrumentation()
  ],
});
```

## Meta Packages for Web

To leverage the most common instrumentations all in one you can simply use the
[OpenTelemetry Meta Packages for Web](https://www.npmjs.com/package/@opentelemetry/auto-instrumentations-web)

## Instrumentation with Browser Extension

> If you'd like to get a quick preview on how your website (or any other site) would look like with OpenTelemetry

Reviewer: Maybe a bit confusing wording to some people as the instrumentation doesn't change how the website looks(well unless it breaks it :P). Maybe mention that this is about previewing what data is collected.

Reviewer 2: ```suggestion
If you'd like to quickly preview what instrumentation would look like with your website (or any other site)
```

Author: Yeah, I was not able to phrase this properly, thanks for the suggestion!
